scrapers/DesperateAmateurs.yml

```diff
@@ -0,0 +1,35 @@
+name: "DesperateAmateurs"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - desperateamateurs.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="title_bar"]/text()
+      Details: //div[@class="gallery_description"]/text()
+      Performers:
+        Name: //a[@class="model_category_link"][contains(@href, 'sets.php')]/text()
+      Tags:
+        Name: //a[@class="model_category_link"][contains(@href, 'category.php')]/text()
+      Image:
+        selector: //img[@class="thumbs"]/@src
+        postProcess:
+          - replace:
+            - regex: (.+image:\s+")(.+jpg)(.+)
+              with: $2
+            - regex: ^
+              with: "https://desperateamateurs.com/"
```

Comment:
the image has the same issue with above it matches all the thumbnails / 16 images
using this selector instead
```yaml
        selector: //img[@class="thumbs" and contains(src,"001.jpg")]/@src
```
only matches the first thumbnail
Another simpler alternative is using
```yaml
      Image: //meta[@name="twitter:image"]/@content
```
or 
```yaml
      Image: meta[@property="og:image"]/@content
```
Note that the meta use a different image each and usually not the same with the video player ( but neither does the 1rst thumb)

Reply:
This makes sense. Thanks so much for checking this over.